packages/form-js-viewer/test/spec/render/components/form-fields/Checklist.spec.js

```diff
@@ -0,0 +1,232 @@
+import {
+  fireEvent,
+  render
+} from '@testing-library/preact/pure';
+
+import Checklist from '../../../../../src/render/components/form-fields/Checklist';
+
+import { createFormContainer } from '../../../../TestHelper';
+
+import { WithFormContext } from './helper';
+
+const spy = sinon.spy;
+
+let container;
+
+
+describe('Checklist', function() {
+
+  beforeEach(function() {
+    container = createFormContainer();
+  });
+
+  afterEach(function() {
+    container.remove();
+  });
+
+
+  it('should render', function() {
```

Comment:
Should we have a test case to verify it sets the `checkbox.checked` for the given `value`?